### PR TITLE
Ps migrations filestorage

### PR DESCRIPTION
**Pull request by dshybeka**
Added simple implementations of data stores
 - LocalDataStore - store files locally
 - SeaweedDataStore - store files in seaweed fs

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/store/filesystem/LocalFilePathGenerator.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	public String generate() {
+
+		String uuid = UUID.randomUUID().toString();
+
+		String levelOne = uuid.substring(0, 2);
+		String levelTwo = uuid.substring(2, 4);
+		String levelThree = uuid.substring(4, 6);
+		String tail = uuid.substring(6);
+
+		System.out.println(levelOne + levelTwo + levelThree + tail);
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/store/filesystem/LocalFilePathGenerator.java]: logger, please!

File: src/test/resources/test-application.properties [hunk 2]
```diff
@@ -6,3 +6,5 @@ spring.datasource.platform=postgresql
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.hibernate.ddl=false
+
+datastore.default.path=d:\\epam\\report-portal\\data\
```
Comment (avarabyeu) [inline src/test/resources/test-application.properties]: can we use some temp dir here?

File: src/main/java/com/epam/ta/reportportal/store/filesystem/LocalFilePathGenerator.java (hunk 1 above)
Comment (dshybeka) [reply src/main/java/com/epam/ta/reportportal/store/filesystem/LocalFilePathGenerator.java]: sure, will update, thanks

File: src/test/resources/test-application.properties (hunk 2 above)
Comment (dshybeka) [reply src/test/resources/test-application.properties]: sure, this is test properties file, will update property to datastore.default.path=/data/storage as in main applicaiton.properties.

**Pull request merged**